kytos/core/napps/manager.py

```diff
@@ -90,7 +89,6 @@ def install(self, napp_uri, enable=True):
             dst = self._installed / napp.username / napp.name
             self._create_module(dst.parent)
             shutil.move(str(napp_folder), str(dst))
-            os.chmod(dst, mode=0o755)
```

Comment:
The tmp permission is set considering the user umask. I know this does not force a desired permission, but this is the way all the other folders are being created, including the parents, like the module's directory created by _create_module(). The only folder that was nos created in this way was the one created with mkdtemp().
I was just trying to keep the folder consistent with the others.
You are right, but if some mode is desired to be forced (755 fo ex) we should for chmod on all the other methods too.